src/Flex.php

```diff
@@ -436,6 +436,11 @@ public function install(Event $event = null)
             $this->io->writeError('');
             $this->io->writeError('What about running <comment>composer global require symfony/thanks && composer thanks</> now?');
             $this->io->writeError(sprintf('This will spread some %s by sending a %s to the GitHub repositories of your fellow package maintainers.', $love, $star));
+        }
+
+        $this->io->writeError('');
+        if ($this->downloader->isEnabled()) {
```

Comment:
When is this `true`? Displaying this on every single `composer u` or `composer i` might be too much, don't you think?

Reply:
This is the way I found to disable the message for global update / install.
Perhaps, personally it does not shock me that it is reminded on each install / update. Otherwise we can display it in a post-installation message

Reply:
WDYT @nicolas-grekas ?